docs/tutorials/modeling.ipynb

```diff
@@ -4,665 +4,784 @@
    "cell_type": "markdown",
```

Comment:
<p>Maybe we should add <code>SourceCatalog.to_models()</code>?  Might be useful...</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3216/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>